src/components/RepoBranchesDrawer/RepoBranchesDrawer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+import GitHubStore from '@store/GitHubStore';
+import { BranchItem, RepoItem } from '@store/GitHubStore/types';
+import { Drawer } from 'antd';
+
+export type RepoBranchesDrawerProps = {
+  selectedRepo: RepoItem | null;
+  onClose: (e: any) => void;
```

Comment:
`any` нужно использовать только в крайних случаях, тут можно просто оставить
```suggestion
  onClose: () => void;
```